app/views/pages/home.html.erb

```diff
@@ -1,4 +1,4 @@
 <%= render 'application/products_filter' %>
 <div class="row">
-  <%= render  @products %>
+  <%= render  partial: 'products/product_card', collection: products, as: :product %>
```

Comment:
Wcześniej nie było strony produktu, a **_product** to był partial dla małej karty produktu na stronie głównej. Zmieniłem ją to na **_product_card** , a **_product** to teraz partial na `show` produktu